core/src/test/java/hudson/FilePathTest.java

```diff
@@ -371,6 +411,10 @@ private void checkTarUntarRoundTrip(String filePrefix, long fileSize) throws Exc
         tmpDirPath.child(tarFile.getName()).untar(outDir, TarCompression.NONE);
         assertEquals("Result file after the roundtrip differs from the initial file",
                 new FilePath(tempFile).digest(), outFile.digest());
+
+        List<?> files = tmpDirPath.child(tarFile.getName()).untar2(outDir, TarCompression.NONE);
```

Comment:
Is the wildcard needed?

```suggestion
        List<String> files = tmpDirPath.child(tarFile.getName()).untar2(outDir, TarCompression.NONE);
```